test/functional/admin/world_location_translations_controller_test.rb

```diff
@@ -76,6 +76,26 @@ class Admin::WorldLocationTranslationsControllerTest < ActionController::TestCas
     end
   end
 
+  view_test "the 'View on website' link on the show page goes to the news page" do
+    location = create(:world_location, translated_into: [:fr], name: "France")
+    get :index, params: { world_location_id: location }
+
+    assert_select "a" do |links|
+      view_links = links.select { |link| link.text =~ /View on website/ }
+      assert_match(/https:\/\/www\.test\.gov\.uk\/world\/france\/news/, view_links.first["href"])
+    end
+  end
+
+  view_test "the view buttons for translations link to the new page on the live site" do
+    location = create(:world_location, translated_into: [:fr], name: "France")
+    get :index, params: { world_location_id: location }
+
+    assert_select "a" do |links|
+      view_links = links.select { |link| link.text =~ /view/ }
+      assert_match(/https:\/\/www\.test\.gov\.uk\/world\/france\/news\.fr/, view_links.first["href"])
```

Comment:
I think you can use `assert_equal` here (as we're matching the entire string, rather than just part of it) which will mean the escaping won't be needed.
```suggestion
      assert_equal("https://www.test.gov.uk/world/france/news.fr", view_links.first["href"])
```

Reply:
Thanks for review @brucebolt! So the problem the match gets around is that there is a random cachebust string added as a query param. Can you think of a good way of just matching the beginning of a string? I agree that being able to specify the string as per yours is nice and readable.

Reply:
What do you think of this instead?

`assert_match(/#{Regexp.escape("https://www.test.gov.uk/world/france/news.fr")}/, view_links.first["href"])`

Reply:
I've updated them to the above ^^ as it's the neatest way I can think of still using the string, but let me know if you manage to come up with a nicer way

Reply:
Oh, I forgot about the cachebust string.

This looks like a nice alternative.